src/adapter/src/coord/sequencer/inner/create_continual_task.rs

```diff
@@ -42,7 +42,6 @@ use crate::optimize::{self, Optimize, OptimizerCatalog};
 use crate::session::Session;
 use crate::util::ResultExt;
 
-// TODO(ct): Big oof. Dedup a bunch of this with MVs.
```

Comment:
After factoring bits of it out to share internally, feels like there is less to do here.